integration/spark/src/main/scala/org/apache/spark/sql/optimizer/CarbonOptimizer.scala

```diff
@@ -23,11 +23,10 @@ import scala.collection.JavaConverters._
 import scala.collection.mutable.ArrayBuffer
 
 import org.apache.spark.sql._
-import org.apache.spark.sql.catalyst.CatalystConf
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.catalyst.expressions.aggregate._
```

Comment:
No @jackylk , it is unused import in Spark 1.5 but it is used in Spark 1.6. All aggregation related code is moved to this package in Spark 1.6.